views/account/account-main.ejs

```diff
@@ -3,29 +3,63 @@
 <div class="main-box">
   <%-include('../partials/account-navigation')%>
   <div class="info-box">
-    <div class="info-box-div account-first-row">more user info</div>
-    <div class="info-box-div">Player stats </br>
-      um dolor sit amet consectetur, adipisicing elit. Non inventore aut
-      nesciunt neque expedita molestias, dolor nisi labore enim laboriosam quos
-      quae aliquam? Et accusantium ducimus maiores ullam tempore ea. general
-      stats: Lorem ipsum dolor sit amet consectetur, adipisicing elit. Non
-      inventore aut nesciunt neque expedita molestias, dolor nisi labore enim
-      laboriosam quos quae aliquam? Et accusantium ducimus maiores ullam tempore
-      ea. general stats: Lorem ipsum dolor sit amet consectetur, adipisicing
-      elit. Non inventore aut nesciunt neque expedita molestias, dolor nisi
-      labore enim laboriosam quos quae aliquam? Et accusantium ducimus maiores
-      ullam tempore ea. general stats: Lorem ipsum dolor sit amet consectetur,
-      adipisicing elit. Non inventore aut nesciunt neque expedita molestias,
-      dolor nisi labore enim laboriosam quos quae aliquam? Et accusantium
-      ducimus maiores ullam tempore ea. general stats: Lorem ipsum dolor sit
-      amet consectetur, adipisicing elit. Non inventore aut nesciunt neque
-      expedita molestias, dolor nisi labore enim laboriosam quos quae aliquam?
-      Et accusantium ducimus maiores ullam tempore ea. general stats: Lorem
-      ipsum dolor sit amet consectetur, adipisicing elit. Non inventore aut
-      nesciunt neque expedita molestias, dolor nisi labore enim laboriosam quos
-      quae aliquam? Et accusantium ducimus maiores ullam tempore ea. general
-      stats: Lorem
+    <div class="info-box-div account-first-row">
+      <div class="personal-info">
+        <p class="personal-info-item">Name:</p>
+        <p class="personal-info-item">Age:</p>
+        <p class="personal-info-item">Location:</p>
+      </div>
+      <p class="bio">bio</p>
+    </div>
+    <div class="info-box-div">
```

Comment:
Something I noticed before and in some cases didn't comment but I would prefer if you kept the tag name (div) out of the class name. In HTML it's already obvious that it's a div, and if it is not in while reading your CSS spreadsheet, then a more descriptive class name should be used. In that case I would prefer `personal-info-wrapper`.